app/src/main/java/com/xmartlabs/doapp/model/Task.java

```diff
@@ -29,15 +29,15 @@
   @PrimaryKey(autoincrement = true)
   long id;
   @Column
-  String name;
+  String title;
+  @Column
+  String description;
   @Column(typeConverter = LocalDateAdapter.class)
   LocalDate date;
   @Column
-  Double duration;
-  @Column
   boolean isFinished;
-  @ForeignKey
-  User user;
-  @ForeignKey
-  Group group;
+//  @ForeignKey
```

Comment:
because I'll be using it in the next feature, I only commented this for now to do some testing of this feature.